src/data/queries/doiRelatedContentQuery.ts

```diff
@@ -1,12 +1,25 @@
 import { gql, useQuery } from "@apollo/client";
 import { workConnection, workFragment } from "src/data/queries/queryFragments";
-import { QueryData, QueryVar } from "src/data/queries/doiQuery";
+import { QueryData } from "src/data/queries/doiQuery";
+import { QueryVar } from "src/data/queries/searchDoiQuery";
+
+export function buildFilterQuery(variables: QueryVar) {
+  const queryParts = [
+    variables.filterQuery,
+    variables.contributor ? `creators.nameIdentifiers.nameIdentifier:"https://orcid.org/${variables.contributor}"` : '',
```

Comment:
it looks like you have switched from `creators_and_contributors` to only doing `creators`....I think this is not quite right.